libraries/ember/index.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>AppTemplate</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+  </head>
+  <body>
+    <script src="/@embroider/core/vendor.js"></script>
+    <script type="module">
+      import Application from "#src/app";
+      import environment from "#config";
```

Comment:
Here and elsewhere, what do these nonstandard import specifiers that start with `#` mean? What part of the build system makes them work? Could you instead write javascript here, with `./dist/config.js` or similar?

Reply:
This is just how package.json#imports works. 

A super helpful feature when you enable type=module.

Reply:
They are standard: https://nodejs.org/api/packages.html#subpath-imports

Reply:
They are standard in _Node_. Recommend avoiding them for maximum ES and tooling compatibility.

Reply:
Vite, Rollup, TypeScript, etc support them as well -- and this HTML is an input to a node program that emits standard JS for the browser, so I don't see why we draw the line here instead of any reliance on any build tool